spidermon/contrib/actions/email/__init__.py

```diff
@@ -22,7 +22,8 @@ class SendEmail(ActionWithTemplates):
     body_text = None
     body_text_template = None
     body_html = None
-    body_html_template = None
+    # Default value for template
```

Comment:
:lipstick: I’m not sure the comment is really necessary.

Reply:
Alright I will remove the comment :)